spec/static/openapi.yaml

```diff
@@ -512,7 +518,16 @@ components:
         description:
           $ref: "#/components/schemas/description"
           description: A description of the features in the collection
-          example: An address.
+          example: |-
```

Comment:
Just code style, I would use `>-` instead of `|-`

Reply:
Hmm, VS Code shows me an error if I put `|>` any idea why?